src/Nordea/Components/Checkbox.elm

```diff
@@ -182,25 +190,35 @@ view attrs (Checkbox config) =
 
                 Simple ->
                     Css.batch []
+
+        notDisabledSpecificStyling =
+            if config.isDisabled then
+                []
+
+            else
+                [ pseudoClass "hover .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
+                , pseudoClass "focus-within .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
+                , cursor pointer
+                ]
     in
     Html.label
         (css
-            [ display inlineFlex
-            , Css.property "gap" "0.5rem"
-            , alignItems center
-            , cursor pointer
-            , appearanceStyle
-            , position relative
-            , pseudoClass "hover .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
-            , pseudoClass "focus-within .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
-            ]
+            ([ display inlineFlex
+             , Css.property "gap" "0.5rem"
+             , alignItems center
+             , appearanceStyle
+             , position relative
```

Comment:
Will this work? Can probably be applied to the other cases as well

```suggestion
             , position relative
             , withAttribute "disabled" 
                             [ pseudoClass "hover .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
                , pseudoClass "focus-within .nfe-checkbox" [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueMedium) ]
                , cursor pointer
                ]
```

Reply:
Yeah, not sure what's best for that one 🤔